Magento2/Sniffs/Legacy/ModuleXMLSniff.php

```diff
@@ -0,0 +1,91 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+
+namespace Magento2\Sniffs\Legacy;
+
+use PHP_CodeSniffer\Files\File;
+use PHP_CodeSniffer\Sniffs\Sniff;
+use SimpleXMLElement;
+
+/**
+ * Test for obsolete nodes/attributes in the module.xml
+ */
+class ModuleXMLSniff implements Sniff
+{
+    /**
+     * Error violation code.
+     *
+     * @var string
+     */
+    protected $warningCode = 'FoundObsoleteAttribute';
+    
+    /**
+     * @inheritdoc
+     */
+    public function register(): array
+    {
+        return [
+            T_INLINE_HTML
+        ];
+    }
+
+    /**
+     * @inheritDoc
+     */
+    public function process(File $phpcsFile, $stackPtr)
+    {
+        $line = $phpcsFile->getTokens()[$stackPtr]['content'];
+        if (strpos(trim($line), '<module ') === false) {
+            return;
+        }
+        
+        $xml = simplexml_load_string($phpcsFile->getTokensAsString(0, 999999));
```

Comment:
Is that possible to load all token until the token content ending with `/>`?

Also, it would be good to handle possible exceptions for XML operations

Reply:
`getTokenAsString` returns the whole file, as there are no PHP tags inside.